llvm/lib/Support/PrettyStackTrace.cpp

```diff
@@ -39,7 +39,8 @@ using namespace llvm;
 
 static const char *BugReportMsg =
     "PLEASE submit a bug report to " BUG_REPORT_URL
-    " and include the crash backtrace.\n";
+    " and include the crash backtrace, preprocessed "
+    "source, and associated run script.\n";
```

Comment:
'preprocessed source' seems to be something specific to clang but this error is emitted by all sorts of compilers that do not have a preprocessor

Maybe say something more generic like: 'and reproduction instructions'?